openshift-knative-operator/hack/001-serving-namespace-deletion.patch

```diff
@@ -1,31 +0,0 @@
-diff --git a/openshift-knative-operator/cmd/operator/kodata/knative-serving/1.3.0/2-serving-core.yaml b/openshift-knative-operator/cmd/operator/kodata/knative-serving/1.3.0/2-serving-core.yaml
```

Comment:
I don't see this patch in https://github.com/skonto/serving/tree/image_injection_1.3/openshift/patches, maybe it's planned for later but I'd keep track of these removed patches.

Reply:
Ok, thanks!

One note on patches: 
- https://github.com/skonto/serving/blob/image_injection_1.3/openshift/release/manifest-patches/001-serving-namespace-deletion.patch is based on a generated bundles, is there any reason for not doing something like this https://github.com/openshift-knative/eventing-kafka-broker/blob/2f16a913ea82e6b36d0ccc7e4eb5f5deea7c0f1b/openshift/release/generate-release.sh#L10-L11, that would remove the need to change `"serving.knative.dev/release: "v1.3.0"` to `serving.knative.dev/release: "v1.4.0"` in the next release.

Similar comment on other patches since basing them on the origin of the final manifests would benefit from future changes.

Reply:
But that's not really relevant for this PR, just a thought

Reply:
Yeah we could do that I will take a note and do it when we merge the patches. As a side note  https://github.com/skonto/serving/tree/image_injection_1.3/openshift/patches, contains also code patches in [https://github.com/skonto/serving/blob/image_injection_1.3/openshift/release/manifest-patches](https://github.com/skonto/serving/blob/image_injection_1.3/openshift/release/manifest-patches/001-serving-namespace-deletion.patch) it is only manfest patches as they are used in Serverless Operator.